src/performance-methodology/morello-microarchitectural-limitations.md

```diff
@@ -20,7 +20,9 @@ The following microarchitecture limitations arose due to the constrained timelin
   The underlying Armv8.2-A architecture targeted by the initial Neoverse N1 has
   no concept of data-dependent exceptions on stores, and that
   micro-architecture does not have a structure that supports data-dependent
-  exceptions for stores in an efficient way.
+  exceptions for stores in an efficient way, therefore for capability stores an
```

Comment:
This is a bit of a run-on sentence. Perhaps we should break it into two sentences, with “therefore ...” its own sentence?